tests/conftest.py

```diff
@@ -232,6 +232,11 @@ def pytest_addoption(parser):
     parser.addoption("--is_parallel_leader", action="store_true", default=False, help="Is the parallel leader")
     parser.addoption("--parallel_followers", action="store", default=0, type=int, help="Number of parallel followers")
 
+    ############################
+    #   SmartSwitch options    #
+    ############################
+    parser.addoption("--dpu-pattern", action="store", default=None, help="dpu host name")
```

Comment:
Is there a scenario where we wouldn't want to have access to all DPUs on a device? It may be simpler to just grab all DPUs from the testbed file instead of manually specifying them

Reply:
The usage is same as the duthost(switch), if the value is all, it will grab all dpus.
For some common tests shared with regular switch, we don't need any dpus.

Reply:
Hi @theasianpianist, 
It will be convenient for us to run the test when we want to specify one dpu to test. so, it is better to keep it.

Reply:
I that case, what if we set the default value to "all" for this parameter? That way we don't need to change how the tests are called

Reply:
hi @theasianpianist ,
If the default value is all, NPU will also be added into dpuhosts. So, we cannot set the default value to all.

Reply:
Good idea. Done